migration/iD11/postgresql/202406072107_IDEMPIERE-6166.sql

```diff
@@ -0,0 +1,6 @@
+-- IDEMPIERE-6166 PostgreSQL DUAL table with more than one record
+SELECT register_migration_script('202406072107_IDEMPIERE-6166.sql') FROM dual;
+
+CREATE UNIQUE INDEX idx_dual ON dual ( dummy )
+;
+
```

Comment:
I think this - https://pgpedia.info/d/dual-dummy-table.html sounds like a better solution.

Reply:
> I think this - https://pgpedia.info/d/dual-dummy-table.html is a better solution.

Thanks @hengsin 

Implemented your suggestion on commit https://github.com/idempiere/idempiere/pull/2387/commits/b92bf8a000a3272d56636c075c5ab2489a8b9364

I also noticed the same problem is present with the table dbreplicasyncverifier - if the COPY command is executed it ends with multiple rows.

I added a trigger for that table that forbids inserting a second row.